pom.xml

```diff
@@ -228,6 +228,11 @@
                             <artifactId>com.regnosys.rosetta.lib</artifactId>
                             <version>${rosetta.dsl.version}</version>
                         </dependency>
+                        <dependency>
```

Comment:
This is a circular dependency. We can't depend on the `cdm-java` jar as this (`rosetta-maven-plugin`) is used to generate the `cdm-java` in the first place.